usaspending_api/common/logging.py

```diff
@@ -204,3 +218,85 @@ def ensure_logging(
             f"logger={cfg_logger_name}, formatter={cfg_formatter_name}, formatter_class={str(formatter_class)})"
         )
     return logger_to_use
+
+
+# CUSTOM Logging EXPORTER for debugging
+class LoggingSpanProcessor(SpanProcessor):
+    def on_end(self, span: ReadableSpan) -> None:
+        trace_id = span.context.trace_id
+        span_id = span.context.span_id
+        logger = logging.getLogger(__name__)
```

Comment:
I know in other files we have a pattern of defining the `logger` as a file scoped constant. What are your thoughts on making that change here? You could then use that constant here and then also use the Logger instead of a `print()` statement down below on Line 281.

Reply:
That is a good idea. It will keep things consistent and will get rid of the lingering print statement. Plus we will be able to use the logger anywhere throughout the `logging.py` to log stuff.